docs/code_navigation/how-to/adding_lsif_to_workflows.mdx

```diff
@@ -39,15 +39,20 @@ Some indexers will work out of the box by just running them in your project root
 
 ```yaml
 jobs:
-  lsif-go:
+  scip-go:
     # this line will prevent forks of this repo from uploading lsif indexes
     if: github.repository == '<insert your repo name>'
     runs-on: ubuntu-latest
-    container: sourcegraph/lsif-go:latest
+    container: sourcegraph/scip-go:latest
     steps:
       - uses: actions/checkout@v1
       - name: Generate index
-        run: lsif-go
+        working-directory: backend/
+        run: scip-go 
+      - name: Upload index
+        uses: docker://sourcegraph/src-cli:latest
+        # note that the upload command also needs to happen in the same directory!
+        working-directory: backend/
```

Comment:
1. I don't think this step should be called `Upload index` since the step after this involves uploading.
2. I think it would be simpler to use the same instructions for downloading src CLI using `curl` as described elsewhere.